api/internal/handler/route/route.go

```diff
@@ -304,6 +304,14 @@ func (h *Handler) Update(c droplet.Context) (interface{}, error) {
 				return handler.SpecCodeResponse(err), err
 			}
 		}
+	} else {
+		//remove exists script
+		script, _ := h.scriptStore.Get(input.Route.ID)
+		if script != nil {
+			if err := h.scriptStore.BatchDelete(c.Context(), strings.Split(input.Route.ID, ",")); err != nil {
+				log.Warnf("try to delete script %s again", input.Route.ID)
```

Comment:
should we write the error message to log file?

Reply:
sure.but i wrote a wrong log here.